app/Modules/Recruitment/Http/Requests/JobAdvertRequest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Modules\Recruitment\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+use Route;
+
+class JobAdvertRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        $rules = [
+            'title' => ['required'],
+            'image' => ['required']
```

Comment:
The image is not required, you can remove it. Please add the description as well as required field.